src/targets/gpu/target.cpp

```diff
@@ -167,6 +168,8 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         dead_code_elimination{},
         enable_pass(not enabled(MIGRAPHX_DISABLE_REDUCE_FUSION{}), fuse_reduce{}),
         dead_code_elimination{},
+        fuse_concat{},
```

Comment:
Asking the question here more out of curiosity. I assume order doesn't matter here for fuse_pointwise vs fuse_concat? Is there a benefit to swapping order ever around the fuse_reduce?

Reply:
It needs to run after `fuse_pointwise` otherwise there will be no pointwise modules to fuse with. 

> Is there a benefit to swapping order ever around the fuse_reduce?

I dont think so. In either case we will have two kernels that need to be run if these two passes overlap.